src/ecephys_etl/modules/align_timestamps/_schemas.py

```diff
@@ -0,0 +1,92 @@
+from argschema import ArgSchema, ArgSchemaParser
+from argschema.schemas import DefaultSchema
+from argschema.fields import Nested, InputDir, String, Float, Dict, Int, List
+
+
+class ProbeMappable(DefaultSchema):
+    name = String(
+        required=True,
+        help='What kind of mappable data is this? e.g. "spike_timestamps"',
+    )
+    input_path = String(
+        required=True,
+        help="Input path for this file. Should point to a file containing a 1D timestamps array with values in probe samples.",
+    )
+    output_path = String(
+        required=True,
+        help="Output path for the mapped version of this file. Will write a 1D timestamps array with values in seconds on the master clock.",
```

Comment:
I think these `help`s should be `description`s:
https://github.com/AllenInstitute/argschema/blob/c7cf8fb53166fecd1d1b8f562e5b8c0c12f03a03/argschema/utils.py#L237-L260